yandextank/validator/validator.py

```diff
@@ -0,0 +1,159 @@
+import collections
+import importlib
+import os
+import pkgutil
+import sys
+import uuid
+
+import yaml
+from cerberus import Validator
+
+TANK_DIR = os.path.dirname(os.path.dirname(importlib.import_module('yandextank').__file__))
+
+class ValidationError(Exception):
+    pass
+
+
+def load_yaml_schema(directory, filename=None):
+    DEFAULT_FILENAME = 'schema.yaml'
+    name = filename if filename else DEFAULT_FILENAME
+    with open(os.path.join(directory, name), 'r') as f:
+        return yaml.load(f)
+
+
+def load_py(directory, filename=None):
+    DEFAULT_PY_MODULE_NAME = 'schema'
+    name = filename if filename else DEFAULT_PY_MODULE_NAME
+    path = os.path.join(directory, name)
+    schema_module = importlib.import_module(os.path.relpath(path, TANK_DIR).replace('/', '.'))
+    return schema_module.SCHEMA
+
+
+def load_schema(directory, filename=None):
+    try:
+        return load_yaml_schema(directory, filename)
```

Comment:
what about json and toml? maybe make one function that takes filename and returns dict. Use it for schemas and configs. And file types can be json, toml, yaml, .py.